plugins/aztec/.env.example

```
@@ -26,3 +26,5 @@ COINBASE=
 
 #Logging verbosity for aztec container (error|warn|info|debug|trace)
 LOG_LEVEL=info
+
+GOVERNANCE_PROPOSER_PAYLOAD_ADDRESS=0xDCd9DdeAbEF70108cE02576df1eB333c4244C666
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)